src/utilities/__tests__/extendSchema-test.js

```diff
@@ -338,6 +338,34 @@ describe('extendSchema', () => {
     expect(printExtensionNodes(someScalar)).to.deep.equal(extensionSDL);
   });
 
+  it('extends scalars by adding specified directive', () => {
+    const schema = buildSchema(`
+      type Query {
+        foo: Foo
+      }
+
+      scalar Foo
+
+      directive @foo on SCALAR
+    `);
+    const extensionSDL = dedent`
+      extend scalar Foo @foo
+
+      extend scalar Foo @specified(by: "https://first.com/foo_spec")
```

Comment:
@IvanGoncharov Thank you for the link to [`UniqueDirectivesPerLocation.js`](https://github.com/graphql/graphql-js/blob/master/src/validation/rules/UniqueDirectivesPerLocation.js#L59-L60).  That is indeed working as expected, and verified by this new test case I just added: d0606579

Apologies for misremember during today's working group call, but the actual situation that I was concerned about is demonstrated here, where a scalar type is extended multiple times.

I just wanted to call this out and to get confirmation that this was valid, and the expected result, as well as ask for set of yes on this specific part of the [implementation](https://github.com/m14t/graphql-js/blob/specified-directive/src/utilities/extendSchema.js#L330-L346)

Thanks again.